cupy/random/_generator_api.pyx

```diff
@@ -320,8 +325,32 @@ class Generator:
             <numpy.random.generator.Generator.poisson>`
         """
         cdef ndarray y
+        cdef ndarray lam_arr
+
+        if not isinstance(lam, ndarray):
+            if type(lam) in (float, int):
+                lam_a = ndarray(1, numpy.float64)
```

Comment:
```suggestion
                lam_a = ndarray((), numpy.float64)
```

`standard_gamma` has the same issue.  `cupy.random.default_rng(0).standard_gamma(3.7).shape` returns `(1,)` but it should be `()`.

Reply:
fixed! thanks :)

Reply:
How about adding `shape=None` parameter to `TestDistributionsPoisson` (and in `TestDistributionsStandardGamma`)?

Reply:
ok, lets add it!

Reply:
Added for both in legacy/generator APIs, legacy throws a different error.